test/e2e/warnings_test.go

```diff
@@ -24,15 +30,18 @@ func TestWarningsFlag(t *testing.T) {
 	logger := Logger{}
 	kapp := Kapp{t, env.Namespace, env.KappBinaryPath, Logger{}}
 	crdName := "test-no-warnings-crd"
-	crName := "test-no-warnings-cr"
+	crName1 := "test-no-warnings-cr1"
+	crName2 := "test-no-warnings-cr2"
 	cleanUp := func() {
-		kapp.Run([]string{"delete", "-a", crName})
+		kapp.Run([]string{"delete", "-a", crName1})
+		kapp.Run([]string{"delete", "-a", crName2})
 		kapp.Run([]string{"delete", "-a", crdName})
 	}
 
 	cleanUp()
 	defer cleanUp()
 	customWarning := "example.com/v1alpha1 CronTab is deprecated; use example.com/v1 CronTab"
+	outputWarning := fmt.Sprintf("\n%sWarning:%s %s", yellowColor, resetColor, customWarning)
```

Comment:
kapp has a way to disable colors (--color...). does it work with warnings?

Reply:
These warnings are generated by [client-go](https://github.com/kubernetes/client-go/blob/f6ce18ae578c8cca64d14ab9687824d9e1305a67/rest/warnings.go#L125), where in the unicode characters are used to add colour. I think we will only be able to disable them entirely and not just for the tests. We can take a look at the --color option to see how it's working.

Reply:
I just tried it out. It does not seem to work for warnings as the warnings are not generated by us.

Reply:
We might be able to use the configure warnings method to disable the colour for warnings whenever the flag is set to false as they both are being used in the same package.

Reply:
hmm... i vaguely remember also having support for NO_COLOR or something like that (https://github.com/vmware-tanzu/color/commit/ba5534bd301c4471f2a82eb42ad3ed966a6361d5)

Reply:
i wonder if for warnings we shouldnt use color. at least for now.

Reply:
We figured out a way to make the `--color` flag work. Wanna have a look at that?

Reply:
Made warnings respect the `FORCE_COLOR` env variable as well